src/layout/index.js

```diff
@@ -5,7 +5,7 @@ import Modal from './Modal'
 import Popover from './Popover'
 import SpacedList from './SpacedList'
 import TextHighlight from './TextHighlight'
-import { ModalPicker } from './Pickers'
+import { ModalPicker, ExternalModalPicker } from './Pickers'
```

Comment:
Bogus import?

Reply:
Sorry, forgot to take that out - this is what I had before:

https://github.com/smartprocure/contexture-react/pull/255/files/b0e1bd3bb5a6a5fb984fb5b99d63d869427b0812

My idea was to just export a `ModalPicker` component that doesn't include its own button and state, so it can be managed... `External`ly 🙃

Then Sam pointed out that this was probably unnecessary overhead so I threw in a commit to show the difference if it were inlined in `FilterList` instead. And yeah, I guess it is kind of unnecessary... I still kinda like it, but neither of us feels strongly about it. What do you think?

Reply:
I like less components because I have to come up with less names. Also Modal + Anything is not really worth exporting in a generic library like this imo since that composition is trivial to do.